django/cantusdb_project/main_app/views/chant.py

```diff
@@ -1312,11 +1299,20 @@ def get_context_data(self, **kwargs):
                 "ghisp": "All",
                 "page": page,
             }
-            page = requests.get(
-                "https://cantusindex.org/search",
-                params=p,
-                timeout=5,
-            )
+            try:
```

Comment:
Since I don't see a change to this template, I'm curious what the user will see if this View fails?

Reply:
In the situation where we hit an SSLError or Timeout on the first page, `if len(cantus_id) == 0` will be True, so `context["results"]` will be set to `[["No results", "No results", "No results"]]`. If we hit an error on subsequent pages, we just `break` out of the `for` loop, and we only display the results we've parsed so far in previous passes through the `for`.

Do you think this needs a comment?

Reply:
No, I think fine for the same reason below.